internal/orchestrion/_integration/internal/trace/span.go

```diff
@@ -159,3 +160,137 @@ func printableSpanAttribute(attrs map[string]any, key string) string {
 	}
 	return fmt.Sprintf("%+v", val)
 }
+
+// FromSimplified parses a trace tree from a simplified string format, with the following criteria:
```

Comment:
This is very similar to snapshot testing.
There are existing libraries that worth checking out: https://github.com/bradleyjkemp/cupaloy and https://github.com/gkampitakis/go-snaps.

I'm not suggesting to use them if there is no benefit.

I like how simple this is!

Reply:
Nice! I think this is slightly different to snapshot testing, though I think we should probably introduce it (using any of those libraries) in dd-trace-go, specially in the contrib tests.